app/controllers/ExclusionListController.scala

```diff
@@ -454,7 +454,7 @@ class ExclusionListController @Inject()(
 
     Logger.info(
       s"[ExclusionListController][searchResultsHandleValidResult] Committing Exclusion for scheme ${request.empRef.toString}" +
-        s", with employees Optimisitic Lock: ${excludedIndividual.get.perOptLock}"
+        s", with employees Optimistic Lock: ${excludedIndividual.get.perOptLock}"
```

Comment:
I know you didn't technically change this bit, but a .get on an optional value is always asking for trouble.

Reply:
^ changed now